interop/kotlinx-metadata/build.gradle.kts

```diff
@@ -16,14 +16,24 @@
 
 tasks.jar {
   manifest {
-    attributes("Automatic-Module-Name" to "com.squareup.kotlinpoet.metadata.specs")
+    attributes("Automatic-Module-Name" to "com.squareup.kotlinpoet.metadata")
+  }
+}
+
+tasks.compileTestKotlin {
+  kotlinOptions {
+    freeCompilerArgs = listOf("-Xjvm-default=all")
   }
 }
 
 dependencies {
+  api(libs.autoCommon)
+  api(libs.guava)
```

Comment:
Any way we can get guava off the classpath?

Reply:
Not easily but we can at least make them implementation deps

![image](https://user-images.githubusercontent.com/1361086/134033860-c4bde675-9c5f-4b5f-82c4-8819528d44d9.png)

Reply:
f4b001b

Reply:
Yeah, that's better, assuming the types don't appear in the ABI